docs/quickstart/installation-guide-for-beginners.mdx

```diff
@@ -115,59 +115,69 @@ If you see a version number, Homebrew has been successfully installed.
 
     c. Wait for the installation to complete. You'll know it's done when you see the command prompt again.
 
 This is the end of the one-time installations and the following steps will be required every time:
 
-    #### 4. Setting up the cluster
+    ### Setting up the cluster
     Now that you have Kind installed, let's create a Kubernetes cluster:
 
-    1. Create a new cluster by running:
+    1. In the Terminal, copy and paste the following command and press `Enter`:
 
-      ```bash
-      kind create cluster --name odigos-demo
-      ```
-    
-    This command creates a new Kubernetes cluster named "odigos-demo".
+    ```bash
+    kind delete cluster --name odigos-demo
+    ```
+
+    This removes any existing test environment with the same name. If this is your first time, you'll see a message saying the cluster doesn't exist, which is fine.
 
-    2. Verify that the cluster is running:
+    2. Create a new cluster by running:
 
-      ```bash
-      kubectl cluster-info --context kind-odigos-demo
-      ```
-    
-      This should display information about your cluster, including the Kubernetes master and CoreDNS.
+    ```bash
+    kind create cluster --name odigos-demo
+    ```
+
+    This command creates a new Kubernetes cluster named "odigos-demo".<br />
+    You'll see a lot of text appear in the Terminal. This is normal. Wait until you see a message that says "Your Kubernetes control-plane has initialized successfully!" and the command prompt appears again.
+
+    3. Verify that the cluster is running:
 
-    #### 5. Install the demo application
+    ```bash
+    kubectl cluster-info --context kind-odigos-demo
+    ```
+
+    This should display information about your cluster, including the Kubernetes master and CoreDNS.
+
+    ### Install the demo application
     With our Kubernetes cluster running, we'll now deploy a demo application consisting of 5 microservices written in Go, Java, Python, .NET, and Node.js:
 
     1. Deploy the application using the following command:
 
-      ```bash
-      kubectl apply -f https://raw.githubusercontent.com/odigos-io/simple-demo/main/kubernetes/deployment.yaml
-      ```
-    
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/odigos-io/simple-demo/main/kubernetes/deployment.yaml
+    ```
+
     This command applies the Kubernetes configuration file that defines our demo application.
     2. Wait for all the pods to be in the 'Running' state:
 
-      ```bash
-      kubectl get pods --watch
-      ```
-   
+    ```bash
+    kubectl get pods --watch
+    ```
+
     Press Ctrl+C to exit the watch mode once all pods are running.
-    
+
     3. Verify that all services are running:
 
-      ```bash
-      kubectl get services
-      ```
+    ```bash
+    kubectl get services
+    ```
+
     You should see a list of services including frontend, inventory, membership, coupon, and pricing.
 
-    To access the demo application:
+    ### To access the demo application:
```

Comment:
Shouldn't this be part of the existing `### Install the demo application`? e.g. number 4

Reply:
Hmm I had the same thought while editing, should I remove this title and increment the subjects #'s below it?

Reply:
I support it :)